.github/workflows/npm.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and publish npm package
+
+on:
+  workflow_dispatch:
+    branches: main
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "latest"
+          registry-url: "https://registry.npmjs.org/"
+
+      - name: Install dependencies
+        run: npm install
+        working-directory: npm/
+
+      - name: Build the project
+        run: npm run build
+        working-directory: npm/
+
+      - name: Publish package
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
This package is currently in my personal npmjs account. Happy to add other collaborators/move it to a loam org instead.

Reply:
Definitely! But let's try with yours for now.  I'm sure if you have the permissions but if you dm the secret I can add it.